modules/navier_stokes/src/userobjects/CNSFVGreenGaussSlopeReconstruction.C

```diff
@@ -114,12 +114,12 @@ CNSFVGreenGaussSlopeReconstruction::reconstructElementSlope()
   for (unsigned int is = 0; is < nside; is++)
   {
     unsigned int in = is + 1;
+    const Elem * neig = elem->neighbor_ptr(is);
 
     /// for internal side
 
-    if (elem->neighbor(is) != NULL)
+    if (neig != NULL && this->hasBlocks(neig->subdomain_id()))
```

Comment:
Minor: Feel free to switch `NULL` -> `nullptr` when you see them or when you touch lines that they are on.